pages.de/common/kubectl-delete.md

```diff
@@ -0,0 +1,32 @@
+# kubectl delete
+
+> Lösche Kubernetes-Ressourcen.
+> Weitere Informationen: <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#delete>.
+
+- Löschen eines bestimmten Pods:
+
+`kubectl delete pod {{pod_name}}`
```

Comment:
```suggestion
`kubectl delete pod "{{pod_name}}"`
```

Optional fix ([source](https://github.com/tldr-pages/tldr/blob/main/pages/common/kubectl-delete.md)): placeholder can be quoted if it can contain any special characters to POSIX-compatible shell.

Reply:
If your pod name contains special characters, you're doing it wrong.